.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "developers.home-assistant",
+    "dockerFile": "Dockerfile",
+    "appPort": [
+        3000
+    ],
+    "postCreateCommand": "yarn install",
+    "extensions": [
+        "davidanson.vscode-markdownlint",
+        "eamodio.gitlens",
+        "editorconfig.editorconfig",
+        "github.vscode-pull-request-github",
```

Comment:
It is only needed for reviewing? And you would need to be authenticated anyway?
Is it really needed to develop/contribute to this project?

Reply:
As above; let's drop

Reply:
Still in?

Reply:
My bad, fixed now